Makefile

```diff
@@ -49,6 +49,9 @@ install-mitie:
 install-full: install install-mitie
 	poetry install -E full
 
+install-docs:
+	cd newdocs/ && yarn install
+
```

Comment:
let's also update the other make targets that work with the docs, e.g. https://github.com/RasaHQ/rasa/pull/6211/files#diff-b67911656ef5d18c4ae36cb6741b7965R86-R90